sci-electronics/kicad/kicad-4.0.0_rc1.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit bzr cmake-utils fdo-mime flag-o-matic gnome2-utils python-r1 vcs-snapshot wxwidgets versionator
+
+DESCRIPTION="Electronic Schematic and PCB design tools."
+HOMEPAGE="http://www.kicad-pcb.org"
+
+SERIES=$(get_version_component_range 1-2)
+MY_PV=$(replace_version_separator 3 '-')
+
+LIB_REVISION="bd0ec6a20c7cc0556093f60eac723e4505947784"
+DOC_REVISION="0d0ed7b9de3afbba5d51fb6d7a25834000ed1099"
+I18N_REVISION="bf793d30d64f4e75ea7a109ea434c1a27926b158"
+
+SRC_URI="https://launchpad.net/${PN}/${SERIES}/${MY_PV}/+download/${PN}-${MY_PV}.tar.xz
```

Comment:
Use of these variables for the SRC_URI are clearly demonstrated in the devmanual [1]. It is confusing for us users to read such comments. Can we agree that this is a stylistic suggestion rather than a technical fix?

[1] https://devmanual.gentoo.org/ebuild-writing/variables/index.html#src_uri